sample-queries/sample-queries.json

```diff
@@ -1845,6 +1845,204 @@
       "tip": "This query requires the TI indicator external id. To find the ID, you can run: GET https://graph.microsoft.com/beta/security/tiIndicators?$top=5",
       "skipTest": false
     },
+    {
+      "id": "d4eb9650-719c-4c42-a124-c26549b671e7",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get articles",
```

Comment:
`list articles`

naming rule of thumb:
* singular item by key/id: "Get"
* a list/collection of items: "List"

Reply:
Please apply to all